shared/gh/js/bootstrap.calendar.js

```diff
@@ -172,16 +210,34 @@ define(['gh.core', 'moment', 'clickover'], function(gh, moment) {
      * @private
      */
     var refreshCalendar = function(ev, evt) {
```

Comment:
This didn't change but it might be good to reword `evt` to `data` as we do in most other places (I think)